websockets/extended-payload-length.any.js

```diff
@@ -1,13 +1,5 @@
-<!doctype html>
-<title>WebSockets : Boundary-value tests for the 'Extended payload length' field in RFC6455 section5.2 'Base Framing Protocol'</title>
-<meta name=timeout content=long>
-<script src=/resources/testharness.js></script>
-<script src=/resources/testharnessreport.js></script>
-<script src=constants.js?pipe=sub></script>
-<meta name="variant" content="">
-<meta name="variant" content="?wss">
```

Comment:
@jdm what would you output though for the worker case? I'd know for https://github.com/w3c/wpt-tools/blob/6bc43a7c185137ccdfdfe114528e43784fc94c43/serve/serve.py#L115 which also needs changing. I filed https://github.com/w3c/wpt-tools/issues/185 since @Ms2ger said he'd get to it.